include/wx/listbase.h

```diff
@@ -617,6 +621,8 @@ typedef void (wxEvtHandler::*wxListEventFunction)(wxListEvent&);
 #define wxEVT_COMMAND_LIST_COL_DRAGGING        wxEVT_LIST_COL_DRAGGING
 #define wxEVT_COMMAND_LIST_COL_END_DRAG        wxEVT_LIST_COL_END_DRAG
 #define wxEVT_COMMAND_LIST_ITEM_FOCUSED        wxEVT_LIST_ITEM_FOCUSED
+#define wxEVT_COMMAND_LIST_ITEM_CHECKED        wxEVT_LIST_ITEM_CHECKED
+#define wxEVT_COMMAND_LIST_ITEM_UNCHECKED      wxEVT_LIST_ITEM_UNCHECKED
```

Comment:
These aliases are in the "old names" section. I don't think such aliases should be added for new event names.